src/views/virtualmachines/tree/utils/utils.tsx

```
@@ -471,6 +473,12 @@ export const filterItems = (item: TreeViewDataItem, input: string) => {
   }
 };
 
+export const getEffectiveShowEmptyProjects = (hasVMs: boolean, showEmptyProjects: string): string =>
+  hasVMs ? showEmptyProjects : SHOW;
```

ESLint

[error] Replace `hasVMs:·boolean,·showEmptyProjects:·string):·string·=>⏎··hasVMs·?·showEmptyProjects·:·SHOW` with `⏎··hasVMs:·boolean,⏎··showEmptyProjects:·string,⏎):·string·=>·(hasVMs·?·showEmptyProjects·:·SHOW)`

(prettier/prettier)